### PR TITLE
docs: add some methods for model documentation

### PR DESCRIPTION
**Pull request by Angelii**
### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Enhancement (changes that improvement of current feature or performance)
- [ ] Refactoring (changes that neither fixes a bug nor adds a feature)
- [ ] Test Case (changes that add missing tests or correct existing tests)
- [ ] Code style optimization (changes that do not affect the meaning of the code)
- [x] Docs (changes that only update documentation)
- [ ] Chore (changes that don't modify src or test files)

### Self Check before Merge

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING**](https://github.com/antvis/x6/blob/master/CONTRIBUTING.md) document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: sites/x6-sites/docs/api/graph/model.en.md [hunk 1]
```diff
@@ -74,6 +74,23 @@ addNode(node: Node, options?: AddOptions): Node
...
+添加多个节点到画布，返回该画布。
+
+<span class="tag-param">参数<span>
+
+| 名称             | 类型                  | 必选 | 默认值  | 描述                                                                |
+|------------------|-----------------------|:----:|---------|-------------------------------------------------------------------|
+| nodes             | (Node.Metadata \| Node)[] |  ✓   |         | [节点元数据](/en/docs/api/model/node#constructor)或[节点实例](/en/docs/api/model/node)数组。   |
+| options.silent   | boolean               |      | `false` | 为 `true` 时不触发 `'node:added'` 和 `'cell:added'` 事件和画布重绘。 |
+| options.sort     | boolean               |      | `true`  | 是否按照  `zIndex` 排序。                                            |
+| options...others | object                |      |         | 其他自定义键值对，可以在事件回调中使用。                              |
```
Comment (NewByVector) [inline sites/x6-sites/docs/api/graph/model.en.md:92]: 这里是不是 options.others?

File: sites/x6-sites/docs/api/graph/model.en.md [hunk 2]
```diff
@@ -74,6 +74,23 @@ addNode(node: Node, options?: AddOptions): Node
 | options.sort     | boolean               |      | `true`  | 是否按照  `zIndex` 排序。                                            |
 | options...others | object                |      |         | 其他自定义键值对，可以在事件回调中使用。                              |
 
+### addNodes(...)
+
+```sign
+addNodes(nodes: (Node.Metadata | Node)[], options?: AddOptions): Graph
+```
+
+添加多个节点到画布，返回该画布。
```
Comment (NewByVector) [inline sites/x6-sites/docs/api/graph/model.en.md:83]: 还可以加上：批量添加节点的时候，推荐使用这个方法，相比多次 addNode，它具备更好的性能。

**Review by NewByVector — CHANGES_REQUESTED** [review]

File: sites/x6-sites/docs/api/graph/model.en.md (hunk 1 above)
Comment (Angelii) [reply sites/x6-sites/docs/api/graph/model.en.md:92]: 遵循类似部分的写法，done ✅

File: sites/x6-sites/docs/api/graph/model.en.md (hunk 2 above)
Comment (Angelii) [reply sites/x6-sites/docs/api/graph/model.en.md]: 谢谢文瑀师兄指点，done ✅ ✅

**Review by NewByVector — APPROVED** [review]

**Pull request merged**